platform/terraform/main.tf

```diff
@@ -84,6 +84,45 @@ resource "azurerm_cosmosdb_sql_database" "cosmosdb-container" {
   resource_group_name = azurerm_resource_group.resource-group.name
 }
 
+resource "random_password" "sql_admin_password" {
+  length = 16
+  special = true
+}
+
+resource "azurerm_key_vault_secret" "platform-sql-admin-username" {
+  name         = "ebis-sql-admin-username"
+  value        = "${var.environment-prefix}-ebis-sql-admin"
```

Comment:
This feels over kill? do we need env prefix on the user name?

Reply:
Will remove it

Reply:
Still showing?